docs/model_developers/release.rst

```diff
@@ -36,42 +36,59 @@ When done, first use git to add files:
 
     git add *
 
-To commit, use the module tool. This will create a new dev release.
+Create a new dev release:
```

Comment:
I think we can improve this further by not using the word "release" here. Perhaps "version"? Otherwise, further down, the line "Once the dev version is ready for release" becomes quite ambiguous.